python/pyspark/sql/dataframe.py

```diff
@@ -473,6 +473,23 @@ def isStreaming(self) -> bool:
         """
         return self._jdf.isStreaming()
 
+    def isEmpty(self) -> bool:
+        """Returns ``True`` if this :class:`DataFrame` is empty.
+
+        .. versionadded:: 3.3.0
+
+         Examples
+        --------
+        >>> df_empty = spark.createDataFrame([], 'a STRING')
+        >>> df_non_empty = spark.createDataFrame([("a")], 'STRING')
+
```

Comment:
No blank lines necessary. Otherwise, the example section in the document build from the docstring is split.
![isEmpty-example-split](https://user-images.githubusercontent.com/4736016/140441029-b81a6f2e-b0bd-4b78-ab0d-490af45f3b9d.png)

Please confirm the layout of the document by `make html` in `python/docs`.
See also http://spark.apache.org/docs/latest/api/python/development/index.html

Reply:
Done. Thanks I am kinda new to this.